crates/jit/src/instantiate.rs

```diff
@@ -120,6 +120,19 @@ impl<'data> RawCompiledModule<'data> {
             None
         };
 
+        // Notify jitprofiling support of published code.
```

Comment:
Is this perhaps stuck around from an old merge conflict?

Reply:
> @jlb6740 it looks like the new crate here is pulling in bindgen at build time I think, right? That's a pretty hefty dependency and one we'd ideally rather avoid. Would it be possible to check in the generated bindings from bindgen via a scheme like [pepyakin/binaryen-rs#52](https://github.com/pepyakin/binaryen-rs/pull/52) perhaps?

Hi @alexcrichton, thanks for the comments. The profiling facilitated by a C library (https://github.com/intel/ittapi) for instrumentation and tracing and so there is a sys crate that uses bindgen to create the rust interface.  Right now I think that crate isn't properly guarded on a windows build so that needs to be corrected, but it works well on linux. Yes .. we can make any update ... I am trying to digest the change here: https://github.com/pepyakin/binaryen-rs/pull/52/files. Did you remove the build time dependence on bindgen .. do it ahead of time and put the resulting file in bindings.rs? Is that what I am looking at there?

Reply:
@jlb6740 right yeah, I moved it to a `dev-dependencies` section so it's still verified that the bindings are correct on `cargo test`, but otherwise the generated bindings are checked into the tree so `bindgen` isn't needed at build time, only when testing the ittapi-rs crate